### DISCUSSION TITLE
metal cutting simulation

### DISCUSSION THREAD

**Discussion by chuang05** [idaholab/moose#28895] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Does moose support modeling the process of metal cutting? any suggestions?

**Comment by GiudGiud** [comment]
Hello
I have not seen such simulations performed with moose

**Reply by chuang05** [reply]
Thank you very much.
